dev/sg/internal/run/command.go

```diff
@@ -30,7 +30,7 @@ type Command struct {
 	Preamble string `yaml:"preamble"`
 
 	ExternalSecrets map[string]secrets.ExternalSecret `yaml:"external_secrets"`
-
+	Description     string                            `yaml:"description"`
```

Comment:
```suggestion
	Description     string                            `yaml:"description"`

```

we might want to preserver the new line here